zoltar-tests/src/main/java/com/spotify/zoltar/IrisExtras.java

```diff
@@ -1,25 +0,0 @@
-/*-
- * -\-\-
- * zoltar-tests
- * --
- * Copyright (C) 2016 - 2018 Spotify AB
- * --
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *      http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * -/-/-
- */
-
-package com.spotify.zoltar;
-
-public final class IrisExtras {
```

Comment:
@rav this is needed because during release a `javadoc.jar` is needed for sub-module `zoltar-tests` (sonatype requires this jar). The only way i got it to generate was by adding this dummy class.